src/GoodsList.tsx

```diff
@@ -7,10 +7,16 @@ type Props = {
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
-        {good.name}
-      </li>
-    ))}
+    {goods.map(good => {
+      const style: React.CSSProperties = {
+        color: good.color,
+      };
+
+      return (
+        <li key={good.id} data-cy="good" style={style}>
+          {good.name}
+        </li>
```

Comment:
don't see any sense to create a separate object for only one style 
```suggestion
      return (
        <li 
           key={good.id} 
           data-cy="good" 
           style={{ color: good.color }}
          >
           {good.name}
        </li>
```